bootstrap/config/config.go

```diff
@@ -668,7 +665,7 @@ func (cp *Processor) waitForCommonConfig(configClient configuration.Client) erro
 	isConfigReady := false
 	isCommonConfigReady := false
 	for cp.startupTimer.HasNotElapsed() {
-		commonConfigReady, err := configClient.GetConfigurationValue(config.CommonConfigDone)
+		commonConfigReady, err := configClient.GetConfigurationValueByFullPath(fmt.Sprintf("%s/%s", configReadyPath, config.CommonConfigDone))
```

Comment:
This is appending `config.CommonConfigDone` each iteration.

Reply:
fixed